src/main/java/com/cloudbees/jenkins/plugins/kubernetes_credentials_provider/SecretUtils.java

```diff
@@ -54,7 +57,10 @@
 
     /** Annotation prefix for the optional custom mapping of data */
     private static final String JENKINS_IO_CREDENTIALS_KEYBINDING_ANNOTATION_PREFIX = "jenkins.io/credentials-keybinding-";
-    
+
+    /** Optional annotation containing a comma separated list of job folders this credential is available to */
+    static final String JENKINS_IO_CREDENTIALS_SCOPE_LABEL = "jenkins.io/credentials-scope";
```

Comment:
a scope most normally global or system.  I think this will be confusing

Reply:
maybe rather `jenkins.io/credentials-folder` then?